src/citrine/informatics/design_spaces/product_design_space.py

```diff
@@ -11,56 +12,75 @@
 
 
 class ProductDesignSpace(Resource['ProductDesignSpace'], DesignSpace):
-    """[ALPHA] An outer product of univariate dimensions, either continuous or enumerated.
+    """[ALPHA] An outer product of design spaces.
+
+    Factors can be other design spaces and/or univariate dimensions.
 
     Parameters
     ----------
     name:str
         the name of the design space
     description:str
         the description of the design space
+    subspaces: List[Union[UUID, DesignSpace]]
+        the list of subspaces to combine, either design spaces defined in-line
+        or UUIDs that reference design spaces on the platform
     dimensions: list[Dimension]
         univariate dimensions that are factors of the design space; can be enumerated or continuous
 
     """
 
     _response_key = None
 
-    uid = properties.Optional(properties.UUID, 'id', serializable=False)
-    name = properties.String('config.name')
-    description = properties.Optional(properties.String(), 'config.description')
-    dimensions = properties.List(properties.Object(Dimension), 'config.dimensions')
-    typ = properties.String('config.type', default='Univariate', deserializable=False)
-    status = properties.String('status', serializable=False)
-    status_info = properties.Optional(
-        properties.List(properties.String()),
-        'status_info',
-        serializable=False
-    )
-    archived = properties.Boolean('archived', default=False)
-    experimental = properties.Boolean("experimental", serializable=False, default=True)
-    experimental_reasons = properties.Optional(
-        properties.List(properties.String()),
-        'experimental_reasons',
-        serializable=False
+    subspaces = properties.List(properties.Union(
+        [properties.UUID, properties.Object(DesignSpace)]
+    ), 'config.subspaces', default=[])
+    dimensions = properties.Optional(
+        properties.List(properties.Object(Dimension)), 'config.dimensions'
     )
+    typ = properties.String('config.type', default='ProductDesignSpace', deserializable=False)
+    # Product design spaces may not be embedded in other subspaces, hence status is required
+    status = properties.String('status', serializable=False)
 
     # NOTE: These could go here or in _post_dump - it's unclear which is better right now
     module_type = properties.String('module_type', default='DESIGN_SPACE')
     schema_id = properties.UUID('schema_id', default=UUID('6c16d694-d015-42a7-b462-8ef299473c9a'))
 
-    def __init__(self,
+    def __init__(self, *,
```

Comment:
Note that this is a breaking change.  I like it (and this is an alpha route), but we'll want to be sure to message it to users.